presto-tests/pom.xml

```diff
@@ -338,7 +339,14 @@
             <scope>runtime</scope>
             <optional>true</optional>
         </dependency>
-        
+
+        <dependency>
+            <groupId>org.apache.ratis</groupId>
+            <artifactId>ratis-metrics-default</artifactId>
+            <version>${dep.ratis.version}</version>
```

Comment:
@sumi-mathew please remove the version from here and add this dependency to the DependencyManagement section in the root pom.xml, alongside the other `org.apache.ratis` dependencies.

https://github.com/prestodb/presto/blob/b8354095ae35f46a0f7e6f5fc2399d544d5ba24d/pom.xml#L2003-L2061